mlflow/tensorflow/__init__.py

```diff
@@ -752,6 +754,22 @@ def autolog(
         warnings.warn("Could not log to MLflow. TensorFlow versions below 1.12 are not supported.")
         return
 
+    def _extract_tensor_as_numpy_input_example_slice(next_input):
```

Comment:
For maintainability, can we add docstrings to the internal methods that we're defining throughout this PR?

Reply:
added to all of them.

Reply:
Thanks!